### DISCUSSION TITLE
Command permissions

### DISCUSSION THREAD

**Discussion by Nazunawa** [jagrosh/MusicBot#841] [Questions] [answered]
i noticed that pause and unpausing requires the DJ role. is there a way to let anyone pause, unpause, and some other commands to use without DJ role?

**Comment by MichailiK** [comment]
Set the DJ role to @everyone.

**Reply by Nazunawa** [reply]
but i dont want everyone to be able to access some commands, such as forceremove, movetrack, forceskip while also being able to pause

**Reply by MichailiK** [reply, marked as answer]
You currently can't specify which commands to restrict in which ways, so this is impossible.

**Reply by Nazunawa** [reply]
oh
